nestkernel/conn_builder.h

```diff
@@ -240,6 +246,12 @@ class AllToAllBuilder : public ConnBuilder
   {
   }
 
+  bool
+  is_symmetric() const
+  {
+    return true;
+  }
```

Comment:
@heplesser Good catch, I did not think of that. Is there any easy way to determine if all-to-all uses randomized synapse parameters? I am thinking of a solution where this functions first checks if there are any randomized parameters involved (if so throws an `BadProperty`) and (otherwise) then returns `true`.